packages/fast-components-styles-msft/src/paragraph/index.ts

```diff
@@ -0,0 +1,18 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { IParagraphClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { fontWeight } from "../utilities/fonts";
+
+const styles: ComponentStyles<IParagraphClassNameContract, IDesignSystem> = {
+    paragraph_1: {
+        fontWeight: `${fontWeight.semilight}`
```

Comment:
nit: not necessary to use backticks for these

Reply:
these are numbers, so I need the backticks for strings.